neo/IO/Helper.cs

```diff
@@ -43,6 +54,17 @@ public static T[] AsSerializableArray<T>(this byte[] value, int max = 0x1000000)
             }
         }
 
+        public static unsafe T[] AsSerializableArray<T>(this ReadOnlySpan<byte> value, int max = 0x1000000) where T : ISerializable, new()
```

Comment:
Any particular reason to use max in hex?

Reply:
It follow the other methods